browser/ipfs/test/BUILD.gn

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2021 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import("//brave/build/config.gni")
+import("//brave/components/ipfs/buildflags/buildflags.gni")
+import("//testing/test.gni")
+
+source_set("brave_browser_ipfs_unittests") {
+  testonly = true
+  if (ipfs_enabled) {
```

Comment:
This is not needed since we only include this into deps with ipfs_enabled check.